src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,34 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  const userExist = user
+    ? <UserInfo user={user} />
+    : 'User does not exist';
+
+  const doneOrNot = classNames(
+    'TodoInfo',
+    { 'TodoInfo TodoInfo--completed': completed },
+  );
```

Comment:
```suggestion
  const doneOrNot = classNames(
      'TodoInfo',
      {
        'TodoInfo--completed': todo.completed,
      },
    )
```

in the curly braces you only need to left class that will appear only for some condition